test/test_other.py

```diff
@@ -6253,6 +6253,37 @@ def test_failing_growth_2gb(self):
     self.run_process([EMXX, '-O1', 'test.cpp', '-sALLOW_MEMORY_GROWTH'])
     self.assertContained('done', self.run_js('a.out.js'))
 
+  def test_failing_growth_wasm64(self):
```

Comment:
We've added other 2GB+ memory-using tests in `browser` since those run sequentially - maybe this is best added there too?

See 

https://github.com/emscripten-core/emscripten/blob/137fea518f015409bfafeff81740e27b85828bd7/test/test_browser.py#L5473-L5478

Reply:
True, but we also have some here too.. see the test above.

I agree we should run this test in the browser too though.

Reply:
Added a browser test too.